### DISCUSSION TITLE
Security Vulenrability Disclosure

### DISCUSSION THREAD

**Discussion by cyberbharathi** [tabler/tabler#919] [Q&A]
Hello Anyone Could guide me on how to Report a Security Vulnerability Report to your Repo? I need to Report a Security Bug in this Repo!

**Comment by sagegwatkin** [comment]
@codecalm do you have an email address @cyberbharathi could contact you about this privately on?

**Comment by rjd22** [comment]
@cyberbharathi hmm, I'm not sure there is a system in place atm. could you send me an email?

me [a] codersopinion [d] com
